odoo/tests/common.py

```diff
@@ -2665,3 +2665,17 @@ def _is_matching(test_filter):
             return True
 
         return False
+
+
+class MockValidator:
```

Comment:
This naming is incorrect, it doesn't validate anything. This sort of behaviour is usually called a matcher e.g. https://github.com/hypothesis/h-matchers